ui/components/data-sources/DataSourcesListItem.js

```diff
@@ -1,35 +1,68 @@
-import React from "react";
+import React, { Component } from "react";
+
 import {
     ListView,
     DropdownKebab,
     MenuItem
 } from "patternfly-react";
 
-const DataSourcesListItem = ({ item }) => {
-    const { type, name } = item;
-
-    return (
-        <ListView.Item
-            className="ds-list-item"
-            heading={type}
-            description="---"
-            leftContent={(
-                <span className="list-item-name">{name}</span>
-            )}
-            actions={(
-                <div>
-                    <DropdownKebab id="data-source-list-item-dropdown" pullRight>
-                        <MenuItem>
-                            Edit Data Source
-                        </MenuItem>
-                        <MenuItem>
-                            Delete Data Source
-                        </MenuItem>
-                    </DropdownKebab>
-                </div>
-            )}
-        />
-    );
-};
-
-export { DataSourcesListItem };
+import { graphql } from "react-apollo";
+import DeleteDataSource from "../../graphql/DeleteDataSource.graphql";
+import GetDataSources from "../../graphql/GetDataSources.graphql";
+
+class DataSourcesListItem extends Component {
```

Comment:
Just a comment: unless you need to make use of the State, you don't need a class component. In this case, better stick with the functional component.